src/app/containers/Metadata/index.test.jsx

```diff
@@ -477,3 +483,22 @@ shouldMatchSnapshot(
     openGraphType="website"
   />,
 );
+
+describe('MetadataContainer', () => {
+  afterEach(() => {
+    isOnClient = true;
+  });
+
+  it('should render the classname attribute as js when onClient is true', async () => {
+    await renderMetadataToDocument();
+    const htmlEl = document.querySelector('html');
+    expect(htmlEl.getAttribute('classname')).toEqual('js');
+  });
+
+  it('should render the classname attribute as no-js when onClient is false', async () => {
+    isOnClient = false;
```

Comment:
So when `isOnClient` is `false`, this means that there is no window being generated and that's what determines if JS is either enabled or disabled?

Reply:
correct

Reply:
Okay cool, is that behaviour the same with all browsers?

Reply:
@HarveyPeachey just tested this on ie, chrome and FF, it has the same behaviour